### PR TITLE
Create kubernetes_build scenario, migrate build jobs

### PR DESCRIPTION
**Pull request by fejta**
ref #1227 #1225 

* Create scenarios/kubernetes_build.py
* Update all build jobs to use this scenario.

In a subsequent PR will:
* migrate the jobs to use a .yaml config instead of .sh 
* migrate timeout inside kubernetes_build.py

### CODE REVIEW

File: jobs/ci-kubernetes-cross-build.sh [hunk 1]
```diff
@@ -19,15 +19,16 @@ set -o pipefail
...
 
 readonly testinfra="$(dirname "${0}")/.."
-
-### job-env
-export KUBE_FASTBUILD=false
-export KUBE_GCS_RELEASE_SUFFIX=-cross  # to avoid colliding with 'kubernetes-build'
+readonly scenario='kubernetes_build.py'
+readonly scenario_args=(
+  --suffix=-cross
+  # --slow
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-cross-build.sh]: do you need something to match `export KUBE_FASTBUILD=false`?

File: scenarios/kubernetes_build.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        '--suffix', help='Append suffix to the upload path if set')
+    PARSER.add_argument(
+        '--unstable',
+        action='store_true',
+        help='Use the unstable debian channel')
+    PARSER.add_argument(
+        '--federation',
+        help='Enable federation with the specified project')
+    PARSER.add_argument(
+        '--kops', help='Upload cops to the specified gs:// path')
```
Comment (krzyzacy) [inline scenarios/kubernetes_build.py]: nit: cops?

File: scenarios/kubernetes_build.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    ARGS = PARSER.parse_args()
+    main(
+        ARGS.script,
+        ARGS.fast,
+        ARGS.release,
+        ARGS.suffix,
+        ARGS.unstable,
+        ARGS.federation,
+        ARGS.kops)
```
Comment (krzyzacy) [inline scenarios/kubernetes_build.py]: maybe worth create a struct for all the args if it keeps growing

**Comment by krzyzacy** [comment]
maybe also worth create an overall documentation about how the new things will work

File: scenarios/kubernetes_build.py (hunk 2 above)
Comment (fejta) [inline scenarios/kubernetes_build.py]: Done

File: scenarios/kubernetes_build.py (hunk 3 above)
Comment (fejta) [inline scenarios/kubernetes_build.py]: Done

File: jobs/ci-kubernetes-cross-build.sh (hunk 1 above)
Comment (fejta) [inline jobs/ci-kubernetes-cross-build.sh]: We will always set `KUBE_FASTBUILD=(true|false)` this is just a reminder for the future that we explicitly want to build all platforms (as opposed to others which may not care and just want the default, if we switch it to `store_false`)

**Comment by fejta** [comment]
Added documentation

**Comment by krzyzacy** [comment]
/lgtm

it's a good idea to make a separate python script, I was trying to jam everything into `bootstrap.py`,  this would be a better pattern.

**Pull request merged**